cdap-docs/examples-manual/source/examples/wikipedia-data-pipeline.rst

```diff
@@ -88,6 +96,28 @@ This application demonstrates:
 .. include:: _includes/_building-starting-running.txt
 
 
+Deploying the Example
+=====================
+Since deploying the WikipediaPipelineApp involves loading an artifact and creating an application from it, the
+preferred method of deploying it is to use the CDAP CLI.
+
+- Load the Artifact::
+
+    $ cdap-cli.sh load artifact target/WikipediaPipeline-3.3.0-SNAPSHOT.jar name WikipediaPipelineApp version

```

Comment:
does this command work? Looks like the command takes config-file before name and version, not sure if CLI does varargs type of args.  You could also leave out name and version and let the command derive them from the filename.